### ISSUE TITLE
Investigate BCEDiceLoss

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/KaggleClouds#1]
In one of the example kernels the author uses `smp.utils.losses.BCEDiceLoss(eps=1.)`. What is Dice loss and how does it differ from the Binary Cross Entropy Loss we're using?

The metric for this competition is Dice so it may be better to optimize for the metric itself.

**Comment by JoshVarty** [comment]
Can get it from: https://github.com/qubvel/segmentation_models.pytorch/blob/master/segmentation_models_pytorch/utils/losses.py

We should compare it (before and after) with standard Binary Cross Entropy loss
